### DISCUSSION TITLE
How to prevent a Property  from being modified by user-submitted values during insertions and updates?

### DISCUSSION THREAD

**Discussion by hooliy** [mikro-orm/mikro-orm#5315] [Q&A]
How to prevent a Property (updateAt,createAt) from being modified by user-submitted values during insertions and updates, similar to the behavior of @CreateDateColumn, @UpdateDateColumn, and @DeleteDateColumn in TypeORM?

**Comment by B4nan** [comment]
Why don't you just validate your input? If you don't want some properties editable, just throw in case they get to the API payload, this is not a responsibility of an ORM.
